src/vendor/bbauth/web.config.hbs

```diff
@@ -1,17 +1,17 @@
 <?xml version="1.0"?>
 <!-- Web.Config Configuration File -->
 <configuration>
-  {{#if isEnabled }}
   <appSettings>
-    <add key="AuthAddress" value="{{ authAddress }}" />
-    <add key="RootUrl" value="{{ rootUrl }}" />
-    <add key="ClientId" value="{{ clientId }}" />
+    {{#if bbauth.isEnabled }}
```

Comment:
This is the check we had in here before right? Does this get generated from bbauth, or is this a stache.yml config? Just curious.